pkg/providers/instance/resperrorhandlers.go

```diff
@@ -97,14 +97,20 @@ func defaultResponseErrorHandlers() []responseErrorHandler {
 func handleLowPriorityQuotaError(ctx context.Context, provider *DefaultProvider, instanceType *corecloudprovider.InstanceType, zone, capacityType string, err error) error {
 	// Mark in cache that spot quota has been reached for this subscription
 	provider.unavailableOfferings.MarkSpotUnavailableWithTTL(ctx, SubscriptionQuotaReachedTTL)
-	log.FromContext(ctx).Error(err, "")
+	log.FromContext(ctx).WithValues(
+		"instanceType", instanceType.Name,
+		"capacityType", capacityType,
+	).Error(err, "low priority quota reached")
```

Comment:
Great to add values for context 👍 Not totally sure about adding a message (here and elsewhere), I suspect the reason it was omitted is that the error itself basically contains it, but maybe does not hurt, given that we don't quite have control over err, but have control over our message. Would be interested in opinion of others.